### PR TITLE
Release notes for backported 0.12.4

### PR DESCRIPTION
**Pull request by davidhewitt**
I released pyo3 `0.12.4` to crates.io already; this PR just pulls the release notes into master.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fix FFI definitions for `PyObject_Vectorcall` and `PyVectorcall_Call`. [#1287](https://github.com/PyO3/pyo3/pull/1285)
 - Fix building with Anaconda python inside a virtualenv. [#1290](https://github.com/PyO3/pyo3/pull/1290)
 
+## [0.12.4] - 2020-11-28
+- Fix reference count bug in implementation of `From<Py<T>>` for `PyObject`, a regression introduced in PyO3 0.12. [#1297](https://github.com/PyO3/pyo3/pull/1297)
```
Comment (alex) [inline CHANGELOG.md:49]: Does this result in a security issue (double free/UAF I guess)? If so, let's submit an advisory over at https://github.com/RustSec/advisory-db

Comment (davidhewitt) [reply CHANGELOG.md:50]: 100% agree. I'm about to call it a day, so will submit in the morning unless you beat me it! 👍

Comment (alex) [reply CHANGELOG.md:50]: I'll go knock it out. Thanks!

Comment (alex) [reply CHANGELOG.md:50]: https://github.com/RustSec/advisory-db/pull/495

**Review by kngwyu — APPROVED** [review]
Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (davidhewitt) [reply CHANGELOG.md:50]: Thanks! 👍

**Pull request merged**